### ISSUE TITLE
`ObjectSchema` keys are unconditionally added to the JSON object validated 

### ISSUE THREAD

**Issue by pluemer** [faceair/jio#2]
```go
package main

import (
	"bytes"
	"log"

	"github.com/faceair/jio"
)

func main() {
	orig := []byte(`{}`)
	data := []byte(`{}`)
	schema := jio.Object().Keys(jio.K{
		"foo": jio.String(),
	})

	if _, err := jio.ValidateJSON(&data, schema); err != nil {
		log.Fatal(err)
	}

	if !bytes.Equal(orig, data) {
		log.Printf("Want %s, got %s.", orig, data)
	}
}
```

prints

```log
Want {}, got {"foo":null}.
```

**Comment by faceair** [comment]
Thanks, it's been fixed in v1.0.1 version.

**Comment by pluemer** [comment]
Thanks a lot, @faceair. That was quick :slightly_smiling_face:
